Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Need C module to be compiled
+FROM debian:12-slim AS builder
+RUN apt-get update && apt-get install --no-install-suggests --no-install-recommends --yes \
+    python3-venv gcc libpython3-dev libjpeg-dev zlib1g-dev libfreetype6-dev && \
+    python3 -m venv /venv && \
+    /venv/bin/pip install --upgrade pip setuptools wheel
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US:en
+ENV LC_ALL en_US.UTF-8
+
+FROM builder as builder-venv
+COPY requirements.txt requirements.txt
+RUN /venv/bin/pip install --disable-pip-version-check -r requirements.txt
+
+# Using the distroless image led to some issues.
+# Without `/venv/bin/python3` `gradio` module was not found. With it, there were some UTF-8 encoding errors.
+#FROM gcr.io/distroless/python3-debian12
+FROM python:3.11
```

Comment:
This sucks since the installation is 1.5Gb. I couldn't use `distroless` images since I kept hitting module + encoding issues. Will try again.

Worst-case, will try different images (ex. `-slim`) in hopes for a slimmer file size.

Reply:
Down to ~1Gb after using `distroless` image (which dependencies brought over + an array of Google Fonts installed)